parm/config/config.base.emc.dyn

```diff
@@ -383,6 +383,9 @@ if [ $OUTPUT_FILE = "nemsio" ]; then
     export DO_CALC_ANALYSIS="NO"
 fi
 
+# Write analysis files for early cycle EnKF 
+export DO_GFS_ENKF_CALC_INCREMENT="YES"
```

Comment:
This is a picky comment.
Can we instead define this as `DO_CALC_INCREMENT_ENKF_GFS`.
Here are my reasons:
1. The default is `DO_CALC_INCREMENT`
2. If doing this for EnKF regardless of cycle, it could be extended to `DO_CALC_INCREMENT_ENKF`, similar to `FHOUT_ENKF`.
3. If doing this for EnKF and only for the GFS, this could be extended to `DO_CALC_INCREMENT_ENKF_GFS`, similar to `FHMAX_ENKF_GFS`.

Other than that, the changes in this PR are fine.

Your thoughts?

Reply:
I agree to use DO_CALC_INCREMENT_ENKF_GFS to be consistent with other variables.